<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPropCompareBinary</class>
 <widget class="QDialog" name="QPropCompareBinary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binary Compare</string>
  </property>
  <widget class="QGroupBox" name="IDC_STATIC_SHARED_SETTINGS">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>80</y>
     <width>482</width>
     <height>176</height>
    </rect>
   </property>
   <property name="title">
    <string>Shared settings</string>
   </property>
   <widget class="QPushButton" name="IDC_COMPAREBINARY_VIEWSETTINGS">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>36</y>
      <width>340</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>View &amp;Settings...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IDC_COMPAREBINARY_BINARYMODE">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>84</y>
      <width>340</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Binary Mode...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IDC_COMPAREBINARY_CHARACTERSET">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>132</y>
      <width>340</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Character Set...</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="IDC_STATIC_BINARY_FILE">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>12</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhMultiLine</set>
   </property>
   <property name="text">
    <string>Binary File &amp;Patterns:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="IDC_COMPAREBINARY_PATTERNS">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>36</y>
     <width>480</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="IDC_COMPAREBINARY_DEFAULTS">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>360</y>
     <width>140</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Defaults</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
